hack/release.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -e
+set -ex
```

Comment:
I'm not really comfortable with this one -- this will be a _lot_ of output (in an already noisy script) and it'll include passwords in the output itself.  Why not just run with something like `bash -x release.sh` when you need this level of debugging?